examples/fpga/artya7/README.md

```diff
@@ -64,15 +64,26 @@ This will create a directory `build` which contains the output files, including
 the bitstream.
 
 
-Initial memory parameter SRAMInitFile can be given to FuseSoc to specify which .vmem file to load the design with.
-Example use case includes loading coremark.vmem which can be used for performance/power analysis.
+Initial memory parameter `SRAMInitFile` can be given to FuseSoc to specify which .vmem file to load the design with.
+Example use case includes loading `coremark.vmem` which can be used for performance/power analysis.
```

Comment:
Nit: this change should go into the first commit where you add the two lines for the first time.